File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_inspect_model.R
# View specified parameters of the deep learning model
model@parameters

# Examine the performance of the trained model
model  # display all performance metrics

h2o.performance(model, train = TRUE)  # training metrics
h2o.performance(model, valid = TRUE)  # validation metrics

# Get MSE only
h2o.mse(model, valid = TRUE)

# Cross-validated MSE
h2o.mse(model_cv, xval = TRUE)

File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_inspect_model.py
# View specified parameters of the Deep Learning model
model.params

# Examine the performance of the trained model
model  # display all performance metrics

model.model_performance(train=True)  # training metrics
model.model_performance(valid=True)  # validation metrics

# Get MSE only
model.mse(valid=True)

# Cross-validated MSE
model_cv.mse(xval=True)